saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md

```diff
@@ -59,12 +59,12 @@
       - Cognito returns Cognito ID with temporary AWS credentials
   - E.g. Provide (temporary) access to write to S3 bucket using Facebook Login
   - 💡 Good for authenticating B2B & B2C users for web + mobile applications.
-- ***Web Identity Federation*** is an alternative to using Cognito but AWS recommends against it.
+- **Web Identity Federation** is an alternative to Cognito but AWS recommeds Web Identity Federation only for advanced scenarios. Mostly try to use Cognito, because it does most of the behind-the-scenes work with public identity provider services for you.
```

Comment:
Small typo: recommends. I like to point of the differences and much better description.

Reply:
That's great...